iOS_SDK/OneSignalSDK/Source/OneSignalAlertViewDelegate.m

```diff
@@ -73,15 +73,14 @@ + (void)showInAppAlert:(NSDictionary*)messageDict {
 
 @end
 
-
 @implementation OneSignalAlertViewDelegate
 
 NSDictionary* mMessageDict;
 
-/*
- delegateReference exist to keep ARC from cleaning up this object when it goes out of scope.
- This is becuase UIAlertView delegate is set to weak instead of strong
- */
+
+ //delegateReference exist to keep ARC from cleaning up this object when it goes out of scope.
```

Comment:
is there any reason behind changing from block to line comment?

Reply:
shouldn't this be deleted?

Reply:
It is deleted in the major release branch, but it cannot be deleted until we drop iOS 7 support. Changed the comment back